### DISCUSSION TITLE
[Feature request] Archive websites

### DISCUSSION THREAD

**Discussion by xplosionmind** [plausible/analytics#1081] [Anything Else]
In my Plausible dashboard I currently have three out of six websites which are not online anymore. I would love to preserve their visits data, but I’d prefer not to have them in front of me every time I open my dashboard. It would be nice to add an archived section, which still shows the websites, but below the active ones.

**Comment by MaybeThisIsRu** [comment]
I would very much love this too.

**Comment by trecca** [comment]
It would be useful to me as well, I hope it will be evaluated

**Reply by metmarkosaric** [reply]
thanks! we do already have a way to pin the sites you need the most to the top of the list which should help with this use case as well. see https://plausible.io/docs/landing-page#pin-your-most-used-sites-for-quick-access

**Reply by trecca** [reply]
Sure, I already use that feature. But this case is exactly the opposite, I'm talking about some sort of archive to keep the offline sites data and not see them alongside with the other ones that are still live. Thanks
